### PR TITLE
Cleanup redundant codes and typos

### PR DESCRIPTION
**Pull request by Vacant2333**
**Please provide a description of this PR:**
Cleanup redundant codes and typos

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [x] Developer Infrastructure

### CODE REVIEW

File: architecture/environments/operator.md [hunk 1]
```diff
@@ -13,15 +13,15 @@ compiled to [Go
 structs](https://github.com/istio/api/blob/master/operator/v1alpha1/operator.pb.go).
 `IstioOperatorSpec` has pass-through fields to the Helm values.yaml API, but these are additionally validated through
 a [schema](../operator/pkg/apis/istio/v1alpha1/values_types.proto).
-1. [Controller](#k8s-controller) code. The code comprises the K8s listener, webhook and logic for reconciling the cluster
+2. [Controller](#k8s-controller) code. The code comprises the K8s listener, webhook and logic for reconciling the cluster
```
Comment (hanxiaop) [inline architecture/environments/operator.md:16]: There are a lot of README changes, and these sequence numbers should not be changed, nor should the table format. They will automatically be formatted.

**Review by hzxuzhonghu — CHANGES_REQUESTED** [review]
Please see @hanxiaop 's comment. The index is automatically increased

File: pkg/config/analysis/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ The purpose of analyzers is to examine Istio configuration for potential problem
 
 ### 1. Create the code
 
-Analyzers need to implement the Analyzer interface ( in the `galley/pkg/config/analysis` package). They should be created under the analyzers subdirectory, and given their own appropriate subpackage.
+Analyzers need to implement the Analyzer interface ( in the `galley/pkg/config/analysis` package). They should be created under the analyzers' subdirectory, and given their own appropriate subpackage.
```
Comment (hanxiaop) [inline pkg/config/analysis/README.md:9]:
```suggestion
Analyzers need to implement the Analyzer interface ( in the `pkg/config/analysis` package). They should be created under the `analyzers` subdirectory, and given their own appropriate subpackage.
```

File: pkg/config/analysis/analyzers/util/find_errorline_utils.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	// Path templates for different fields with different paths, may edited by future developers if not covered in this list
+	// Path templates for different fields with different paths, may edit by future developers if not covered in this list
```
Comment (hanxiaop) [inline pkg/config/analysis/analyzers/util/find_errorline_utils.go:26]:
```suggestion
	// Path templates for different fields with different paths may be edited by future developers if not covered in this list
```

**Review by howardjohn — CHANGES_REQUESTED** [review]
Please remove all the markdown changes, changes to generated files, changes of variables from `name -> _`

I would encourage you to spend time on issues [that actually matter](https://github.com/istio/istio/issues?q=is%3Aopen+is%3Aissue+label%3A%22community%2Fgood+first+issue%22). Changes like this just waste everyones time.

**Comment by howardjohn** [comment]
Actually I don't think we should do this at all. its too big and has to many changes that don't make sense. Please focus on real issues! We would love to see contributions that help users.

**Pull request closed without merging**